src/i18n.ts

```diff
@@ -69,7 +69,7 @@ locale.subscribe(newLocal => {
 
 await init({
   fallbackLocale: 'en',
-  initialLocale: get(persistantLocale),
+  initialLocale: getLocaleFromQueryString('l') || get(persistantLocale),
```

Comment:
Is this the way around it works in the other apps? Or do settings win over URL?

Reply:
In classroom it's used in the default settings value which isn't persisted unless edited. So it works the other way around.

Reply:
So that might be equivalent to using it as the initial value for the persistantLocale store. Which hopefully doesn't persist its initial value unless edited (looks that way from the code, but please check in incognito).

Reply:
Thanks, good point. I've added `getLocaleFromQueryString('l')` into `initialLocale` (https://github.com/microbit-foundation/ml-trainer/pull/202/commits/7e46eeae51b68c827084ee7ddeb456bde4a34edc). 

https://review-ml.microbit.org/lang-query-string/thenextgen/?l=cy now gives me English in my browser (since I have already picked English in settings > language)

https://review-ml.microbit.org/lang-query-string/thenextgen/?l=cy now gives me Welsh incognito

Reply:
Great!